sdk/generic-sdk-core/generic-core-http-okhttp/src/test/java/com/generic/core/http/okhttp/OkHttpHttpClientHttpClientTests.java

```diff
@@ -41,7 +41,7 @@ protected String getServerUri(boolean secure) {
     }
 
     @Override
-    protected HttpClient createHttpClient() {
-        return new OkHttpHttpClientProvider().createInstance();
+    protected HttpClient getHttpClient() {
+        return new OkHttpHttpClientProvider().getNewInstance();
```

Comment:
I think we should have the tests use the shared instance as that would / should be the default behavior when unspecified